base/special/log.jl

```diff
@@ -26,7 +26,7 @@ import .Base.Math.@horner
   #   print("(",l_hi,",",l_lo,"),")
   # end
 
-const t_log_Float64 = [(0.0,0.0),(0.007782140442941454,-8.865052917267247e-13),
+const t_log_Float64 = ((0.0,0.0),(0.007782140442941454,-8.865052917267247e-13),
```

Comment:
I'm curious about what the benefit of this is. Both versions seem to generate similar code for the access at the moment. At least with a naive proxy for the table load looking like

```julia
const tab = ((1.0,1.0), #= some more pairs =#)
foo(n) = @inbounds tab[n][1]*tab[n][2]
```

Reply:
hm, they shouldn't generate similar code. there's an entire extra pointer load for a vector, not to mention that it can't constant fold it:
```
	movabsq	$4807388912, %rax       ## imm = 0x11E8AEEF0
	movq	(%rax), %rax
```
vs
```
	movabsq	$4627942040, %rax       ## imm = 0x113D8CA98
	movabsq	$4627942056, %rcx       ## imm = 0x113D8CAA8
```

Reply:
Thanks, I see what's going on now.